app.ts

```diff
@@ -0,0 +1,54 @@
+
+function randomInt(start:number,end:number){
+
+    return start + Math.floor(Math.random()*(end-start));
+}
+
+function generate_alphabet(){
+
+    let tmp:string[]=[];
+
+    for (var i=65;i<123;i++)
+    {   
+
+        if (i>=91 && i<=96) {}
+        else{
+
+        tmp.push(String.fromCharCode(i));
+        }
+    }
+
+return tmp;
+}
+
+function generate_password() {
+       var pass: String;
+       pass="";
+       let numbers: number[] = [1,2,3,4,5,6,7,8,9,0];
```

Comment:
You can use `const` here, since you are never reassigning the value of these variables, you can use `let` instead of `var`.
further reading:
https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75
https://basarat.gitbooks.io/typescript/content/docs/let.html